test/lib/ansible_test/_data/requirements/sanity.changelog.in

```diff
@@ -1,3 +1,3 @@
-rstcheck < 4  # match version used in other sanity tests
+rstcheck <= 6.1.0  # match version used in other sanity tests
```

Comment:
You probably have to do
```suggestion
rstcheck >= 6.0.0 <= 6.1.0  # match version used in other sanity tests
```
to avoid having problems with rstcheck < 6 vs. rstcheck 6 compatibility.

(Might need a comma or something, I'm not 100% sure on the syntax right now.)